docs/api/en/renderers/WebGLRenderer.html

```diff
@@ -471,6 +471,16 @@ <h3>[method:null setScissorTest]( [param:Boolean boolean] )</h3>
 			scissor area will be affected by further renderer actions.
 		</p>
 
+		<h3>[method:null setOpaqueSort]( [param:Function method] )</h3>
+		<p>
+		Sets the custom opaque sort function for the WebGLRenderLists. Pass null to use the default painterSortStable function.
+		</p>
+
+		<h3>[method:null setTransparentSort]( [param:Function method] )</h3>
+		<p>
+		Sets the custom transparent sort function for the WebGLRenderLists. Pass null to use the default reversePainterSortStable function. In some cases the transparent materials sort may get rendered in the wrong order. This issue may cause materials to rendere behind other materials instead of the front of them and vice versa.
```

Comment:
>In some cases the transparent materials sort may get rendered in the wrong order. This issue may cause materials to render behind other materials instead of the front of them and vice versa.

I think that can be removed. File a bug report if you want.

Reply:
@WestLangley I'm not sure I understand. I have opened a [bug report](https://github.com/mrdoob/three.js/issues/13889#issuecomment-385225320) and `mrdoob`'s solution was to add this function.

Please advise.

Reply:
I am suggesting you remove those two sentences.

If this is merged, I think it would be a good idea to specify the calling arguments. See how `Obejct3D.onBeforeRender` is documented.

Reply:
@WestLangley How about this:
`[method:null setTransparentSort]( [param:Function method] )`
`Sets the custom transparent sort function for the WebGLRenderLists. Pass null to use the default reversePainterSortStable function.`

Or:
`[method:null setTransparentSort]( [param:Function method] )`
`Sets the custom transparent sort function for the WebGLRenderLists. This function is called with the following parameter: method`

Or you can suggest anything else.

Reply:
>If this is merged, I think it would be a good idea to specify the calling arguments. See how Obejct3D.onBeforeRender is documented.

Actually, what I was referring to was the signature of the function you are passing. Maybe @Mugen87 has a good suggestion for how to handle this...

Reply:
I think a sentence like the following (from `Object3D.onBeforeRender()` is okay:
`This function is called with the following parameters: renderer, scene, camera, geometry, material, group.`

However, the TS declaration should be adjusted. Instead of using `Function`, it's better to  define the callback inline like so:
```ts
onBeforeRender: (
   renderer: WebGLRenderer,
   scene: Scene,
   camera: Camera,
   geometry: Geometry | BufferGeometry,
   material: Material,
   group: Group
) => void;
```

Reply:
However, this can also be done with a separate PR. Maybe it's better to merge this one first and then move on.

Reply:
Personally, I think the OP has a modeling problem. But @mrdoob thinks this is needed, then at least remove the two sentences I referenced prior to merging.

Reply:
>How about this:
[method:null setTransparentSort]( [param:Function method] )
Sets the custom transparent sort function for the WebGLRenderLists. Pass null to use the default reversePainterSortStable function.
>Or:
[method:null setTransparentSort]( [param:Function method] )
Sets the custom transparent sort function for the WebGLRenderLists. This function is called with the following parameter: method
>Or you can suggest anything else.

@WestLangley Can you please write here the exact sentence you want me to push?

Reply:
@Mugen87 can you please guide me on how to complete this PR?

Reply:
The comment https://github.com/mrdoob/three.js/pull/16909/files#r352293599 asked you to remove the following two sentences.

> In some cases the transparent materials sort may get rendered in the wrong order. This issue may cause materials to rendere behind other materials instead of the front of them and vice versa.

Besides, I suggest update the TS declaration file. Instead of using `Function`, use this:
```js
onBeforeRender: (
   renderer: WebGLRenderer,
   scene: Scene,
   camera: Camera,
   geometry: Geometry | BufferGeometry,
   material: Material,
   group: Group
) => void;
```